test/integration/project-management.spec.ts

```diff
@@ -75,6 +81,28 @@ describe('admin.projectManagement', () => {
     });
   });
 
+  describe('setDisplayName()', () => {
+    it('successfully set project\'s display name', () => {
+      const newDisplayName = generateUniqueProjectDisplayName();
+      // TODO(caot): verify that project name has been renamed successfully
```

Comment:
Please add a bit more detail to this TODO - ie. Indicating that this can only be verified after the ability to `get` project metadata is implemented.

Reply:
Done